src/c4/gatherv_pt.cc

```diff
@@ -51,6 +51,8 @@ template void indeterminate_allgatherv<unsigned>(std::vector<unsigned> &outgoing
 
 template void determinate_allgatherv<unsigned>(std::vector<unsigned> &outgoing_data,
                                                std::vector<std::vector<unsigned>> &incoming_data);
+template void determinate_allgatherv<double>(std::vector<double> &outgoing_data,
+                                             std::vector<std::vector<double>> &incoming_data);
```

Comment:
Is there a test in `c4` for this instantiation?  If not, can you add a simple test?

Reply:
That is a good idea - will add a simple test for this instantiation.

Reply:
Some unit tests for these `[in]determinate_allgatherv` instantiations have been added to `tstGatherScatter.cc`

Reply:
Woops - need a `soft_equiv` for the test with `double`.

Reply:
This is fixed now.